packages/ui/src/components/DatePicker/PreviousDateRangePicker.tsx

```diff
@@ -16,7 +16,7 @@ import { Box } from '../Box/Box'
 import { colors } from '../../css/colors'
 import { TimeInput } from './TimeInput'
 
-export { getDefaultPresets } from './utils'
+export { getDefaultPresets, getNow } from './utils'
```

Comment:
Don't see where this is being used 🤔 - could it be cleaned up?

Reply:
this is an export - it's used in a few different places so they reference the same "now" time

Reply:
Oh, sorry, I wasn't looking closely and thinking this was an `import` 😅